process-engine/skeleton
# 👷 Ignore Failures of Docker Cleanup

## What did you change?

- Ignore any Failures of `cleanup_docker();`

Fixes #56

## How can others test the changes?

See jenkins build https://ci.process-engine.io/blue/organizations/jenkins/process-engine_node-lts%2Fskeleton/detail/feature%2Fignore_docker_cleanup_failures/4/pipeline/

## PR-Checklist

Please check the boxes in this list after submitting your PR:
